smart-contracts/genV2/UnlockDiscountTokenV2.patch

```diff
@@ -0,0 +1,124 @@
+--- contracts/UnlockDiscountTokenV2.ref	2021-06-04 18:41:45.209035257 +0200
```

Comment:
This should be put into another branch, specific to the token  upgrade

Reply:
yes removed the entire `genv2` folder. For another branch